.vscode/settings.json

```diff
@@ -5,4 +5,12 @@
     ],
     "rust-analyzer.check.command": "clippy",
     "rust-analyzer.showUnlinkedFileNotification": false,
+    "files.associations": {
```

Comment:
Does pylance need to be added to the devcontainer?

Reply:
Devcontainer already has pylance: [source](https://github.com/spellshift/realm/blob/0ec17b8df5527eabb91178c98610b8885675596c/.devcontainer/devcontainer.json#L22)